profile/README.md

```diff
@@ -43,7 +43,7 @@ The core components of KubeSlice are detailed below with links to their respecti
 * [kubeslice-controller](https://github.com/kubeslice/kubeslice-controller) -  The KubeSlice Controller orchestrates the creation and management of slices on worker clusters. The KubeSlice Controller components and the worker cluster components can coexist on a cluster. Hence, the cluster running the KubeSlice controller can also be used as a worker cluster.
 * [gateway-certs-generator](https://github.com/kubeslice/gateway-certs-generator) - This is an opinionated single-file OpenVPN TLS certificate configuration generator for slice gateways. it is an enhancement to easy-rsa (typically bundled with OpenVPN).
 * [worker-operator](https://github.com/kubeslice/worker-operator) - the KubeSlice Worker Operator is a Kubernetes operator that manages the lifecycle of KubeSlice worker clusters.
-* [kubeslice](https://github.com/kubeslice/kubeslice) - It contains applications provided by Avesha Systems that are ready to launch on Kubernetes using the Kubernetes Helm.
+* [charts](https://github.com/kubeslice/kubeslice) - It contains charts provided by Avesha Systems that are ready to launch on Kubernetes using the Kubernetes Helm.
```

Comment:
@pnavali  - The name of the repo is still kubeslice. So, can we retain `kubeslice` ? 

We can change the `applications` to `charts` in the repo description.